app/controllers/containers_controller.rb

```diff
@@ -48,6 +48,28 @@ def reschedule
     end
   end
 
+  # GET /relocate
+  def relocate
+    @container = Container.find(params[:id])
+    @nodes = Node.where.not(id: @container.node_id)
+  end
+
+  # POST /schedule_relocation
+  def schedule_relocation
+    @container = Container.find(params[:id])
+    node = Node.find(params[:node_id])
+    @container.update_status('SCHEDULE_RELOCATION')
```

Comment:
There's no logic here for checking container relocate to the same node like other API?

Reply:
or maybe we should have a `relocate` method in model so that we are not repeating ourselves?

Reply:
updated mas @giosakti 
we implemented `relocate` method on model like you suggested